misc/python/materialize/checks/cluster.py

```diff
@@ -15,13 +15,26 @@
 
 class CreateCluster(Check):
     def manipulate(self) -> List[Testdrive]:
+        # This list MUST be of length 2.
```

Comment:
@def- This was EXTREMELY painful to figure out. I originally tried to add a new element to this list and tests started to fail in bizarre ways. It took me a long time to figure out that all elements past index 1 are ignored.

Reply:
Oh no, I'll add an assertion for this.